### ISSUE TITLE
operator-sdk up local failed to run

### ISSUE THREAD

**Issue by fanminshi** [operator-framework/operator-sdk#273]
Currently, the 8a1704d319afaf123b67ab0b80ecb6e3c689328f won't allow the `operator-sdk up local` to start the operator due to `WATCH_NAMESPACE` not set:

```sh
$ operator-sdk up local
INFO[0000] Go Version: go1.10
INFO[0000] Go OS/Arch: darwin/amd64
INFO[0000] operator-sdk Version: 0.0.5+git
FATA[0000] Failed to get watch namespace: WATCH_NAMESPACE must be set
exit status 1
Error: failed to run operator locally: exit status 1
```

The above issue is introduced at https://github.com/operator-framework/operator-sdk/pull/227 when we change the behavior of operator getting its namespace value from the env variable instead of a hardcoded value.
 
A quick work around is to define the `WATCH_NAMESPACE` env var: 

`WATCH_NAMESPACE="default" operator-sdk up local`

It might be nice to let the `operator-sdk up local` take in a flag `--namespace` to define where the operator watches the resource and set a default namespace value to `default`.

**Comment by fanminshi** [comment]
cc/ @spahl @hasbro17

**Comment by jhernandezb** [comment]
I can work on this if no one has picked it

**Comment by spahl** [comment]
Go ahead! Thanks!

**Comment by fanminshi** [comment]
fixed via #274

**Comment by gosoon** [comment]
$ export WATCH_NAMESPACE="default"
$ go run cmd/manager/main.go
